src/components/manage/Blocks/Listing/ImageGallery.jsx

```diff
@@ -85,9 +85,19 @@ const ImageGalleryTemplate = ({ items }) => {
       imageSRCThumb = flattenToAppURL(
         item[settings.listingPreviewImageField]?.scales.thumb.download,
       );
-    } else {
+    } else if (item.url) {
       imageSRCOriginal = `${item.url}/@@images/${settings.listingPreviewImageField}/large`;
       imageSRCThumb = `${item.url}/@@images/${settings.listingPreviewImageField}/thumb`;
+    } else if (item.url) {
+      imageSRCOriginal = flattenToAppURL(
+        `${item['@id']}/@@images/${settings.listingPreviewImageField}/large`,
+      );
+      flattenToAppURL(
+        (imageSRCThumb = `${item['@id']}/@@images/${settings.listingPreviewImageField}/thumb`),
```

Comment:
This is not correct, an assignment in a function?

Reply:
yea, obviously... No idea why that did not trigger an error.

Reply:
Fixed. No idea why I added the flattenToAppURL() there in the first place